demo/demo.py

```diff
@@ -56,3 +56,5 @@ class Config:
 # 7      id8  Label_A      1008
 # 8      id9  Label_A      1009
 # 9     id10  Label_B      1010
+
+ParquetResource(path=path, test_path=local_test_path_without_testdir)  # <<- injectable??
```

Comment:
What does this comment mean?

Reply:
Ah, that was just me messing around. At that point I was checking if `test_path` would also be injected. (Which I then added across the board.) The serde classes then check that the incoming serde path called `file` 🙄 - is fully injected.